lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
I don't think the gem side would have changes, because the declaration is only used by the web app.

Or do you think the declaration should be in the gem, and the web app just link to the database using extensions?